pegasus/sites.v3/code.org/public/css/ai-skinny-banner-2023.scss

```diff
@@ -0,0 +1,109 @@
+@import "color", "font", "breakpoints";
+
+.ai-skinny-banner-2023-default {
+  box-sizing: content-box;
+  height: 130px;
+  background: url("/images/banners/ai-bg.png") top center no-repeat;
+  background-size: cover;
+
+  @media screen and (max-width: $width-md) {
+    padding: 0 1.5em;
+  }
+
+  @media screen and (max-width: $width-sm) {
+    padding: 1em 1em 0.5em;
+    height: unset;
+  }
+
+  a:hover,
+  a:active,
+  a:visited,
+  a:link {
+    text-decoration: none;
+  }
+
+  .wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    max-width: 960px;
+    margin: 0 auto;
+
+    @media screen and (max-width: $width-sm) {
+      justify-content: center;
+    }
+  }
+
+  img {
+    margin: 0 2em;
+
+    @media screen and (max-width: $width-md) {
+      margin: 0;
+    }
+
+    @media screen and (max-width: $width-sm) {
+      width: 50px;
+      margin-right: 1.5em;
```

Comment:
Is this supposed to work in right-to-left languages? If so, avoiding `margin-right` in favor of something more directionally agnostic (like `gap`) is preferred

Reply:
Ooo TIL 🙀 — just updated!

Reply:
Nice! You can "force" the page to render right-to-left by changing the `direction` tag in the HTML to `rtl`. When I first did that for the catalog page, I had to change like 7 things 😬 

<img width="366" alt="image" src="https://user-images.githubusercontent.com/9142121/234283821-b67f5531-7fac-4cad-8c41-0c0017fe1c42.png">

Reply:
Haha! Thanks for sharing the knowledge, adding to my notes doc 🙌